core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -64,7 +66,7 @@ private JsonRenderedNode renderJson(PlanRepresentation plan, NodeRepresentation
     {
         private final String id;
         private final String name;
-        private final String identifier;
+        private final Map<String, String> identifier;
```

Comment:
Please improve the commit message to say something like below
```
NodeRepresentation#identifier is updated to be a Map of key-value pairs 
instead of a raw String. This allows JsonRenderer to produce more structured output
while TextRenderer serialises it into a String to keep text format output similar to the current one.
```